docs/developing.md

```diff
@@ -28,6 +28,8 @@ make package
 
 We use a combination of `Makefile` and [Goreleser](https://goreleaser.com/) to automate our release process.
 
+NOTE: Makefile no longer supports building the binary file using Windows on AMD64
+
```

Comment:
This is true, but it's not the only problem. The binary cannot be built for Windows at all now (we use some unix-specific stuff), not only because it's not in Makefile. We can reword this to be more generic. Something like "The load-test tool does not support Windows". And maybe putting somewhere else, not under the `Releasing` section. Even at the beginning of this file. Thoughts, @mattlam88?

Reply:
That makes sense I'll update the PR to include that at the top